horovod/common/operations.cc

```diff
@@ -1541,23 +1542,23 @@ void CheckForStalledTensors(HorovodGlobalState& state) {
     std::chrono::steady_clock::time_point start_at = std::get<1>(m.second);
 
     if (now - start_at > STALL_WARNING_TIME) {
+      std::stringstream message;
       if (!preamble) {
-        std::cerr << "WARNING: One or more tensors were submitted to be "
-                     "reduced, gathered or broadcasted by subset of ranks and "
-                     "are waiting for remainder of ranks for more than "
-                  << std::chrono::duration_cast<std::chrono::seconds>(
+        LOG(WARNING) << "One or more tensors were submitted to be "
```

Comment:
Shall we redirect this into `message` as well?

Reply:
Okay